docs/ref/request-response.txt

```diff
@@ -130,8 +130,9 @@ All attributes should be considered read-only, unless stated otherwise below.
     Available headers depend on the client and server, but here are some
     examples:
 
-    * ``CONTENT_LENGTH`` -- the length of the request body (as a string).
-    * ``CONTENT_TYPE`` -- the MIME type of the request body.
+    * ``CONTENT_LENGTH`` -- The length of the request body (as a string).
+    * ``CONTENT_TYPE`` -- The MIME type of the request body.
+    * ``HTTP_ACCEPT`` -- The HTTP Accept header sent by the client.
```

Comment:
We could describe all these as "The FOO header sent by the client." but that doesn't seem particularly useful. Could you describe it similar to how HTTP_ACCEPT_ENCODING/ACCEPT_LANGUAGE are described?